babel.config.js

```diff
@@ -19,17 +19,7 @@ module.exports = function (api) {
 
   return {
     presets: [
-      isTestEnv && [
-        '@babel/preset-env',
-        {
-          targets: {
-            node: 'current'
-          },
-          modules: 'commonjs'
-        },
-        '@babel/preset-react'
-      ],
-      (isProductionEnv || isDevelopmentEnv) && [
+      [
```

Comment:
@Judahmeek the reason that test is different is for Jest tests that run on Node and not in the browser.

https://rossta.net/blog/why-doesnt-webpacker-use-my-test-config-when-i-run-rails-tests.html